language/move-lang/src/lib.rs

```diff
@@ -162,7 +164,10 @@ pub fn move_compile(
     interface_files_dir_opt: Option<String>,
     sources_shadow_deps: bool,
     flags: Flags,
-) -> anyhow::Result<(FilesSourceText, Result<Vec<CompiledUnit>, Errors>)> {
+) -> anyhow::Result<(
+    FilesSourceText,
+    Result<(Vec<CompiledUnit>, Option<Vec<ModuleTestPlan>>), Errors>,
```

Comment:
Thoughts about making the test plan extraction more manual? And not always given with compile?

Someone can then manually invoke this by doing compile up to CFGIR, building the test plan, and then continuing to bytecode. Thoughts?

Reply:
I really like this idea. Integrating it into the compiler fully feels a bit odd since in most cases you'e just throwing it away at the end.  I'll update thing to do it this way :)